### PR TITLE
Add Metainformation (Tags, References)

### PR DESCRIPTION
**Pull request by pitakill**
# Issue
Fill `meta` of the post

# Solution
Implement the logic to set that info

# Long explanation
The creation of some of the metainformation of the Post needs to be filled different from the creation of the Article. So we need special logic for that kind of information

# Test
screenshot_2018-09-24_18-40-36

### CODE REVIEW

File: articles.go [hunk 1]
```diff
@@ -44,5 +45,61 @@ func articlesLogic() {
...
 		}
+
+		// ID of the created article
+		id := response["id"]
+
+		// Get Data from article to make petitions accordingly
+		metaInfoItems := []string{"tags", "references"}
+
+		for _, meta := range metaInfoItems {
+			metaInfo := data["data"].(map[string]interface{})["attributes"].(map[string]interface{})["meta"].(map[string]interface{})[meta].([]interface{})
```
Comment (gatero) [inline articles.go]: this line is too long 😮

Comment (coderdiaz) [reply articles.go]: Wooooo!

Comment (pitakill) [reply articles.go]: hahah, yeah is nasty!

XD

Comment (pitakill) [reply articles.go]: Does anyone of you know a better approach for split this kind of line? :-P

Comment (gatero) [reply articles.go]: you can do something like:
~~~golang
article := data["data"].(map[string]interface{})
attributes := article["attributes"].(map[string]interface{})
metaInfo := attributes["meta"].(map[string]interface{})[meta].([]interface{})
~~~

maybe have more lines but improves readability

Comment (pitakill) [reply articles.go]: That is not just more lines (which is irrelevant, really); this has more allocation in memory, maybe not a big impact, but a more allocation of memory centainly.

Improves readability (which is good). I'm going to do it that way

**Pull request merged**